packages/wp-gatsby/src/ActionMonitor/ActionMonitor.php

```diff
@@ -297,7 +297,7 @@ function updateUserIsPublic( $post_id, $post ) {
 			? $current_user->data->user_nicename
 			: "User";
 
-		$relay_id = Relay::toGlobalId( 'user', $user_id );
+		$relay_id = Relay::toGlobalId( 'user', strval( $user_id ) );
```

Comment:
@TylerBarnes ID should be a string as is goes into `base64_encode`
See https://github.com/ivome/graphql-relay-php/blob/master/src/Relay.php#L188